collab-document/src/document_data.rs

```diff
@@ -41,8 +41,11 @@ pub fn default_document_data() -> DocumentData {
   let mut text_map: HashMap<String, String> = HashMap::new();
 
   // page block
-  let page_id = generate_id();
-  let children_id = generate_id();
+  let page_id = match Uuid::parse_str(document_id) {
+    Ok(uuid) => uuid_id_from_document_uuid(&uuid, PAGE),
+    Err(_) => generate_id(),
```

Comment:
why do i still need nanoid? Can we use a new uuid directly or just return error from [uuid_id_from_document_uuid](url)